apps/web/.env.template

```
@@ -13,6 +13,9 @@ MUTUAL_CREDIT_WHITELIST_SIGNER_PRIVATE_KEY=
 # On-chain mirror of signal upvotes (optional; off when unset)
 SIGNALS_CONTRACT_ADDRESS=
 SIGNALS_RELAYER_PRIVATE_KEY=
+# Issues/revokes per-space integration keys for external signal ingestion.
+# Ops-only on purpose: public spaces must never expose their write credentials.
+HYPHA_SPACE_API_KEY_OPS_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HYPHA_SPACE_API_KEY_OPS_SECRET key should go before the MUTUAL_CREDIT_WHITELIST_API_KEY key

(UnorderedKey)